compute/Dockerfile.compute-node

```diff
@@ -12,10 +12,26 @@ ARG DEBIAN_FLAVOR=bullseye-slim
 #########################################################################################
 FROM debian:$DEBIAN_FLAVOR AS build-deps
 ARG DEBIAN_FLAVOR
-RUN apt update &&  \
+ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR
```

Comment:
I don't think this line is required, `ARG`  works the same as `ENV` for the changes in the PR

```suggestion
```

Reply:
I copy-pasted it from later in the file, I guess we can remove it there too then